gen-tc.sh

```diff
@@ -126,6 +127,12 @@ case $1 in
 			42586aa50d84870f12244b6c9acdda982e3719ab \
 			"arm-${1}-linux-gnueabi"
 		;;
+	remarkable)
+		Build_CT-NG \
+			https://github.com/tcrs/crosstool-ng.git \
```

Comment:
I'd suggest a PR over by @NiLuJe instead, unless there are reasons not to. ;-)

Reply:
What he said ;).

Reply:
In which case, I'd also fix the CPU (it's an A9), and the glibc's min kernel version (4.1.28).

If they actually updated to 4.9 on current FW releases, I'd be inclined on keeping that to 4.9, but mentioning that fact in the sample's comment ;).

Reply:
I'll do that. I'm not actually sure what the glibc min kernel version is set to for latest firmware version. Is there an easy way to tell from a ssh shell or do you have to look through the SDK?

Reply:
Oops, missed that comment.

The glibc symlinks is usally a good indicator: `0 lrwxrwxrwx 1 root root 12 Dec 12 05:59 /lib/libc.so.6 -> libc-2.30.so`, although it may be horribly lying (hi, Kobo!), so the sure-fire way to check is actually *running* the libc shared library:

```
┌─(niluje@illyria:pts/21)──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────(~/MPLAYER/koxtoolchain)─┐
└─(0.45:75%:19:03:%)── /lib/libc.so.6                                                                                                                                                                                                                                                                                                                   ──(Sun, Feb 02)─┘
GNU C Library (Gentoo 2.30-r3 p4) stable release version 2.30.
Copyright (C) 2019 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.
There is NO warranty; not even for MERCHANTABILITY or FITNESS FOR A
PARTICULAR PURPOSE.
Compiled by GNU CC version 9.2.0.
libc ABIs: UNIQUE IFUNC ABSOLUTE
For bug reporting instructions, please see:
<https://bugs.gentoo.org/>.
```